website/README.md

```diff
@@ -1,25 +1,5 @@
-# Website
+# authentik documentation source
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+This directory contains the source files for the [authentik technical documentation](https://docs.goauthentik.io/docs?utm_source=github) and the [authentik integration guides](https://docs.goauthentik.io/integrations?utm_source=github).
 
-## Installation
-
-```console
-npm ci
-```
-
-## Local Development
-
-```console
-npm run watch
-```
-
-This command starts a local development server and open up a browser window. Most changes are reflected live without having to restart the server.
-
-## Build
-
-```console
-npm run build
-```
-
-This command generates static content into the `build` directory and can be served using any static contents hosting service.
+Contributions are welcome - see our [contributor guidelines](https://docs.goauthentik.io/docs/developer-docs?utm_source=github) for more information.
```

Comment:
please expand this section to say Yes, contributions are welcome but we expect them to locally build and test and then yes send a PR please!

Reply:
@dominic-r is this done?